File: CS/Serialization/MyGridInfoRegistrator.cs
using DevExpress.XtraGrid;
using DevExpress.XtraGrid.Registrator;
using DevExpress.XtraGrid.Views.Base;

namespace Serialization
{
	public class MyGridInfoRegistrator : GridInfoRegistrator
	{
		public override BaseView CreateView(GridControl grid)
		{
			return new MyGridView(grid as GridControl);
		}

		public override string ViewName
		{
			get
			{
				return "MyGridView";
			}
		}
	}
}

File: VB/Serialization/MyGridInfoRegistrator.vb
Imports Microsoft.VisualBasic
Imports System
Imports DevExpress.XtraGrid
Imports DevExpress.XtraGrid.Registrator
Imports DevExpress.XtraGrid.Views.Base

Namespace Serialization
	Public Class MyGridInfoRegistrator
		Inherits GridInfoRegistrator
		Public Overrides Function CreateView(ByVal grid As GridControl) As BaseView
			Return New MyGridView(TryCast(grid, GridControl))
		End Function

		Public Overrides ReadOnly Property ViewName() As String
			Get
				Return "MyGridView"
			End Get
		End Property
	End Class
End Namespace
